<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>661</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>661</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(64, 65, 66)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="button1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(64,65,66);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(33,34,34);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>主页</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoHome"/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="button2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(64,65,66);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(33,34,34);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>音频库</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="button3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(64,65,66);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(33,34,34);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>音频处理</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCut"/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="button4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(64,65,66);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(33,34,34);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>作者寄语</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>208</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>572</width>
        <height>507</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>572</width>
        <height>507</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
